proposals/Wikimedia-and-the-open-source-ecosystem.md

```diff
@@ -0,0 +1,201 @@
+
+
+6. **How might this work be sustained long-term after an initial seed grant?**
+
+Wikimedia's community of editors will maintain this content after the seed grant. Content in the Wikimedia platform is longer lasting than many other digital publications because it is presented for updating as needed, and actively curated by a sizeable community. Beyond that, any institution which wishes to make content contributions to the Wikimedia platform can follow our precedent for doing more or for otherwise building on these projects, either directly or by sponsoring anyone else to contribute further. 
+
+Usually, the most difficult part of organizing good content in the Wikimedia platform is arranging for it to have a good start. When Wikipedia and Wikidata have poor coverage of a popular topic, the usual reasons include that volunteers have been unable to identify suitable information sources to ingest into the Wikimedia platform, and that they lack starting documentation for describing this difficulty to off-wiki experts who would contribute sources if they were asked in a way that fits with their workflows. For strategic development beyond the labor capacity of volunteers, sponsored Wikimedia contributors can develop Wikimedia content on topics of general interest which improves the development environment for everyone. This project will lower the barriers of engagement with the open-source software ecosystem for anyone contributing to Wikimedia projects in the future.
+
+
+### Resources needed
+
+The above presents a list of pilot proposals. We cannot do all of them and would like feedback on which ones we should prioritize to best match with goals of the foundation.
+
+A reasonable scale for any of the projects sketched out here would be about $50,000 per year, all of which will go to personnel who will invoice their labor hours against this. As stated above, the projects can all be rescaled in both directions, e.g. scaled down for prototyping or scaled up for increased impact.
```

Comment:
@Daniel-Mietchen this is a very interesting proposal, thanks for sharing!

One question on my end -- how would this project work with Wikipedia's paid contributions policy?

Reply:
> @Daniel-Mietchen this is a very interesting proposal, thanks for sharing!
> 
> One question on my end -- how would this project work with Wikipedia's paid contributions policy?

Thanks for asking and being aware of the policy. There are several answers:

  1. The Wikimedia community applies the paid contributions policy to editors doing marketing and branding edits for products, organizations, and associated people. In contrast, our proposals increase access to general reference information which would not benefit any company or other legal entity in particular.
  1. We follow the recommendations of the paid contributions policy regardless. Whatever we do would be proposed publicly from the start, transparently performed, and have documented outcomes.
  1. One collaborator, Lane Rasberry, is staff Wikimedian in Residence at the School of Data Science at the University of Virginia, and has experience managing Wikipedia editing through an institution. Lane is also a coordinator for the Wikimedians in Residence Exchange Network, which is a community of Wikipedia editors, each of whom is employed by an expert organization to develop Wikipedia. The most common employers who do this are libraries, museums, universities and research institutes. That network arranges public community review of projects, and we would seek such oversight and support for this project too.

I recognize how strict the paid contributions policy is for most Wikimedia projects, and since you mention it, you must know the need to be careful and to avoid supporting a project contrary to Wikipedia's values. If you like these project ideas and our conversation gets to a point where you would appreciate more Wikipedia community peer review and sign-off, then we can arrange that to the extent of your interest.

Reply:
Thanks!